ci/ci.sh

```diff
@@ -3,9 +3,17 @@ set -xe
 
 # Defaults (in case they are not set by the CI)
 F=${F:-production}
-DC=${DC:-dmd1}
 DIST=${DIST:-xenial}
 
+# Binary/package name deduced based on supplied DMD version
+case "$DMD" in
+    1.*    ) DC=dmd1 ;;
+    2.*.s* ) DC=dmd-transitional ;;
+    *      ) DC=dmd ;;
+esac
+
+apt install -y $DC=$DMD-$DIST
```

Comment:
Although with your current script I think the default would be broken, I think it might be better to default to something that makes sense (here the installed dmd1).